backend/tracim_backend/lib/core/content.py

```diff
@@ -1594,7 +1597,7 @@ def has_pdf_preview(self, revision_id: int, file_extension: str) -> bool:
                 file_path,
                 file_ext=file_extension
             )
-        except UnsupportedMimeType:
+        except Exception:
```

Comment:
I suggest to continue to except `UnsupportedMimeType` (and return None in that case) and except Exception (and log a warning about the error).

Reply:
done